### PR TITLE
parity with iOS

### PR DESCRIPTION
**Pull request by Brianggalvez**
*backgroundColor
*default subtitle to avoid crash
*apply duration in miliseconds

version bump

added dist

### CODE REVIEW

File: android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java [hunk 1]
```diff
@@ -53,6 +58,8 @@ public void show(KrollDict args)
 		Alerter.create(currentActivity)
 			.setTitle(title)
 			.setText(subtitle)
+			.setDuration(duration*1000) //iOS parity. Alerter expects miliseconds
```
Comment (hansemannn) [inline android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java:61]: This should be optional. Also fix spacing

File: android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java [hunk 2]
```diff
@@ -41,7 +41,12 @@ public void show(KrollDict args)
 		int duration = args.getInt("duration");
 		String title = args.getString("title");
 		String subtitle = args.getString("subtitle");
-		int color = TiConvert.toColor(args.getString("color"));
+		
+		if (subtitle == null) {
+			subtitle = "";
```
Comment (hansemannn) [inline android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java:46]: Use `optString("subtitle", "")` instead.

File: android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java [hunk 3]
```diff
@@ -41,7 +41,12 @@ public void show(KrollDict args)
...
+		}
+		
+		int color = TiConvert.toColor(args.getString("backgroundColor"));
```
Comment (hansemannn) [inline android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java:49]: default values

File: android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java [hunk 4]
```diff
@@ -53,6 +58,8 @@ public void show(KrollDict args)
 		Alerter.create(currentActivity)
 			.setTitle(title)
 			.setText(subtitle)
+			.setDuration(duration*1000) //iOS parity. Alerter expects miliseconds
+			.setBackgroundColorInt(color)
```
Comment (hansemannn) [inline android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java:62]: Same here

**Review by hansemannn — CHANGES_REQUESTED** [review]

File: android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java (hunk 4 above)
Comment (Brianggalvez) [reply android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java:62]: Which should be the default values for duration and backgroundColor?

**Comment by hansemannn** [comment]
Superseded by #9 and now released! https://github.com/lambus-platform/titanium-notification-banner/releases/tag/android-3.0.0

**Pull request closed without merging**